src/dms-preview/azext_dms/_help.py

```diff
@@ -5,6 +5,100 @@
 
 from knack.help_files import helps
 
+helps['dms task'] = """
+    type: group
+    short-summary: Manage service-level tasks for a Database Migration Service instance.
+    long-summary: |
+        Service-level tasks are non-migration tasks that are performed at the service level. This is usually for administrative and maintenance work which applies to the service's agent and VM.
+"""
+
+helps['dms task cancel'] = """
+    type: command
+    short-summary: Cancel a service-level Task if it's currently queued or running.
+"""
+
+helps['dms task create'] = """
+    type: command
+    short-summary: Create and start a service-level task.
+    long-summary: |
+        The following tasks are currently supported:
+            1) Check OCI driver against Oracle source to check for compatibility
+            2) Upload OCI driver install package
+            3) Install an already uploaded OCI driver package
+
+    parameters:
+        - name: --task-type
+          type: string
+          short-summary: >
+            The type of task to create. The supported types are CheckOciDriver, UploadOciDriver, and InstallOciDriver.
+        - name: --task-options-json
+          type: string
+          short-summary: >
+            Information and settings relevant to the particular task's type. This can be either a JSON-formatted string or the location to a file containing the JSON object. See examples below for the format.
+          long-summary: >
+            For CheckOciDriver:
+                {
+                    // The version of your Oracle source server against which to check the driver's compatibility.
+                    // Can also be null to get a list of supported versions.
+                    "serverVersion": "Oracle source version"
+                }
+
+            For UploadOciDriver:
+                {
+                    "ociDriverPath": File share path of the driver install package,
+                    "userName": "user name",    // if this is missing or null, you will be prompted
+                    "password": "password"      // if this is missing or null (highly recommended) you will be prompted
+                }
+
+            For InstallOciDriver:
+                {
+                    // The output of the UploadOciDriver task can be used here
+                    "ociDriverPackageName": "Oci driver installer package name"
+                }
+    examples:
+        - name: Create a task which checks if the currently installed OCI driver supports
+          text: >
+            az dms task create -g myresourcegroup --service-name mydms -n checkocitask1 --task-options-json "{\"serverVersion\": \"12.2.0.1\"}" --task-type CheckOciDriver
+"""
+
+helps['dms task delete'] = """
+    type: command
+    short-summary: Delete a service-level Task.
+    parameters:
+        - name: --delete-running-tasks
```

Comment:
We will recommend to use get_three_state_flag() for boolean parameter.

Reply:
May I ask why? The user will either want to delete the running tasks or not. Additionally, this param is used elsewhere in our cli commands. I don't like the idea of having it one way here and another way everywhere else. If it is a requirement for some reason, I think a more prudent way forward is to create a backlog item for fixing this throughout.

Reply:
get_three_state() is cli build-in argument type. It also can provide convenience to users. For more information, please refer to https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#registering-flags.

Reply:
I agree with @vchske in that would be a more extensive update since we use this param in our core cli implementation. It makes sense to make the change throughout as opposed to in bits and pieces.